packages/babel-traverse/src/path/index.js

```diff
@@ -113,6 +113,7 @@ export default class NodePath {
     }
 
     return ourScope;
+    return this.isScope() ? new Scope(this) : scope;
```

Comment:
this isn't intentional right since there's 2 returns?

Reply:
Ha, that's probably why the tests were failing.